website/docs/registry/providers/publishing.html.md

```diff
@@ -0,0 +1,86 @@
+
+The provider repository on GitHub must match the pattern `terraform-provider-{NAME}`, and the repository must be public.
+
+### Documenting your Provider
+
+Your provider should contain an overview document (index.md), as well as a doc for each resource and data-source. See [Documenting Providers](documenting.html) for details about how to ensure your provider documentation renders properly on the Terraform Registry.
+
+### Creating a GitHub Release
+
+Publishing a provider requires at least one version be available on GitHub Releases. The tag must be a valid [Semantic Version](https://semver.org/) preceded with a `v` (for example, `v1.2.3`).
+
+Terraform CLI and the Terraform Registry follow the Semantic Versioning specification when detecting a valid version, sorting versions, solving version constraints, and choosing the latest version. Prerelease versions are supported (available if explicitly defined but not chosen automatically) with a hyphen (-) delimiter, such as `v1.2.3-pre`.
+
+#### Using GoReleaser locally
+
+GoReleaser is a tool for building Go projects for multiple platforms, creating a checksums file, and signing the release. It can also upload your release to GitHub Releases.
+
+1. Install [GoReleaser](https://goreleaser.com) using the [installation instructions](https://goreleaser.com/install/).
+1. Copy the [.goreleaser.yml file](https://github.com/hashicorp/terraform-provider-scaffolding/blob/master/.goreleaser.yml) from the hashicorp/scaffolding provider repository.
+1. Cache the password for your GPG private key with `gpg --armor --detach-sign` (see note below).
+1. Set your `GITHUB_TOKEN` to a [Personal Access Token](https://github.com/settings/tokens) that has the **public_repo** scope.
+1. Tag your version with `git tag v1.2.3`.
+1. Build, sign, and upload your release with `goreleaser release --rm-dist`.
+
+-> GoReleaser does not support signing binaries with a GPG key that requires a passphrase. Some systems may cache your GPG passphrase for a few minutes. If you are unable to cache the passphrase for GoReleaser, please use the manual release preparation process below, or remove the signature step from GoReleaser and sign it prior to moving the GitHub release from draft to published.
+
+#### Manually Preparing a Release
+
+If for some reason you're not able to use GoReleaser to build, sign, and upload your release, you can create the required assets by following these steps, or encode them into a Makefile or shell script.
+
+The release must meet the following criteria:
+
+* There are 1 or more zip files containing the built provider binary for a single architecture
+    * The binary name is `terraform-provider-{NAME}_v{VERSION}`
+    * The archive name is `terraform-provider-{NAME}_{VERSION}_{OS}_{ARCH}.zip`
+    * It's recommended to build for all of the available OS/architecture pairs if possible.
```

Comment:
Let's remove this for now, without the list, this reads like you should build for everything.
```suggestion
```